### DISCUSSION TITLE
Need Help in array

### DISCUSSION THREAD

**Discussion by HirayaEs** [Asabeneh/30-Days-Of-JavaScript#596] [Q&A] [answered]
is there a difference in splice and slice having hard time to understand and new to coding

**Comment by TianMeds** [comment, marked as answer]
The splice() method adds and removes array elements while The slice() method selects from a given start, up to a (not inclusive) given end.
The slice has a syntax of the array.slice(start, end) and for splice array.splice(index, howmany, item1, ....., itemX) you can visit It in the w3school if your a beginner

**Reply by HirayaEs** [reply]
thank you bro

**Reply by ravi-aratchige** [reply]
This is great! Thank you so much.
